ObjectiveGitTests/GTNoteSpec.m

```diff
@@ -0,0 +1,86 @@
+//
+//  GTNoteSpec.m
+//  ObjectiveGitFramework
+//
+//  Created by Slava Karpenko on 2016/05/17.
+//  Copyright (c) 2016 Wildbit LLC. All rights reserved.
+//
+
+#import <Nimble/Nimble.h>
+#import <ObjectiveGit/ObjectiveGit.h>
+#import <Quick/Quick.h>
+
+#import "QuickSpec+GTFixtures.h"
+
+QuickSpecBegin(GTNoteSpec)
+
+__block GTRepository *repository;
+__block GTCommit *initialCommit;
+
+beforeEach(^{
+	NSURL *fileURL = [self.tempDirectoryFileURL URLByAppendingPathComponent:[[NSUUID alloc] init].UUIDString isDirectory:NO];
+	repository = [GTRepository initializeEmptyRepositoryAtFileURL:fileURL options:nil error:NULL];
+	expect(repository).notTo(beNil());
+	
+	GTTreeBuilder *builder = [[GTTreeBuilder alloc] initWithTree:nil repository:repository error:NULL];
+	expect(builder).notTo(beNil());
+	
+	GTTreeEntry *entry = [builder addEntryWithData:[@"Xyzzy" dataUsingEncoding:NSUTF8StringEncoding] fileName:@"test.txt" fileMode:GTFileModeBlob error:NULL];
+	expect(entry).notTo(beNil());
+	
+	GTTree *tree = [builder writeTree:NULL];
+	expect(tree).notTo(beNil());
+	
+	initialCommit = [repository createCommitWithTree:tree message:@"Initial commit" parents:nil updatingReferenceNamed:@"refs/heads/master" error:NULL];
+	expect(initialCommit).notTo(beNil());
+});
+
+it(@"can create notes", ^{
+	// Annotate the commit
+	GTSignature* sig = [repository userSignatureForNow];
+	expect(sig).notTo(beNil());
+	
+	NSError* err = nil;
```

Comment:
Nitpick: misplaced `*`(also on `GTSignature` above), and `NSMutableArray` below.